source/Government.cpp

```diff
@@ -579,6 +587,15 @@ string Government::GetHail(bool isDisabled) const
 	return phrase ? phrase->Get() : "";
 }
 
```

Comment:
As per our [style guide](https://github.com/endless-sky/endless-sky/wiki/C++-Style-Guide#large-vertical-breaks), put three blank lines between functions in .cpp files.
```suggestion



```